url/src/lib.rs

```diff
@@ -1310,9 +1332,23 @@ impl Url {
     ///
     /// ```
     /// use url::Url;
-    /// # use std::error::Error;
     ///
-    /// # fn run() -> Result<(), Box<dyn Error>> {
+    /// # use url::ParseError;
```

Comment:
This is not super pretty. If URL had a `no_std` error type that also supports strings, it could be nicer. However, it's not visible in docs so should be fine

Reply:
Changed this back given that errors_in_core is going to be stable in a week